### PR TITLE
Feature/refactor gnqa UI

### PR DESCRIPTION
**Pull request by Alexanderlacuna**
This Pr;
* Adds the use of htmx to gnqa codebase
* implement new search history page

### CODE REVIEW

**Comment by Alexanderlacuna** [comment]
cc @BonfaceKilz

File: gn2/wqflask/templates/gnqa_answer.html [hunk 1]
```diff
@@ -54,26 +57,24 @@ <h4 class="panel-title" style="font-family: 'Linux Libertine','Georgia','Times',
...
-					    <button type="button"  data-toggle="modal" data-target=".bd-pubmed-modal-lg-{{reference.doc_id}}" style="border:none;text-decoration:underline;outline: none;padding:5px;margin-left:25px;background: transparent;">
-						<span><b>See PubMed Info</b></span>
-					    </button>
-					</div>
-
+					<details >
+					  <summary>See PubMed Info</summary>
+					  <div style="font-family: 'Linux Libertine','Georgia','Times','Source Serif Pro',serif;margin-top:1.4em">
+					    <h3><b>{{ reference.pubmed[0].get('title') }}:</b></h3>			
+					    <p><b>Authors:</b><span class="text-muted">{{ reference.pubmed[0].get('authors') }}</span></p>
```
Comment (BonfaceKilz) [inline gn2/wqflask/templates/gnqa_answer.html:72]: Fix the punctuation of the comma separated list in Python.

File: gn2/wqflask/templates/gnqa_search_history.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    </div>
+  </header>
+  <div class="container row">
+  <div class="panel panel-default col-sm-6 col-sm-offset-3 ">
+    {% for record in prev_queries %}
+    <div class="panel-body">
+      <div class="row">
+	<input name="" type="checkbox" value="" class="col-sm-1" class="text-primary">
+	<div class="col-sm-10">
+	  {% for id,val in  record.items() %}
```
Comment (BonfaceKilz) [inline gn2/wqflask/templates/gnqa_search_history.html:21]: *nitpick*: punctuation/spacing problems.

File: gn2/wqflask/templates/gnqa_search_history.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	    <button
+	      hx-get="/gnqna/hist/search/{{id}}"
+              hx-target="#swap"
+	      hx-swap="innerHTML"
+	      hx-trigger= "click"
+	      data-bs-toggle="tooltip"
+	      data-bs-placement="left"
+	      title="/gnqna/hist/search?{{id}}"
```
Comment (BonfaceKilz) [inline gn2/wqflask/templates/gnqa_search_history.html:29]: Update the search API endpoint in another PR.

**Review by BonfaceKilz — APPROVED** [review]

**Pull request merged**